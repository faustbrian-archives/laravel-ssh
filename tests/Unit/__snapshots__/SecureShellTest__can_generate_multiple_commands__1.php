<?php

return 'ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o BatchMode=yes -o PasswordAuthentication=no -p 22 user@example.com \'bash -se\' << \\EOF-KK-SSH
whoami
cd /var/log
EOF-KK-SSH';
