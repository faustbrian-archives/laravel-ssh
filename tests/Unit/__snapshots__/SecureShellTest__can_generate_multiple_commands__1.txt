ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o BatchMode=yes -o PasswordAuthentication=no user@example.com 'bash -se' << \EOF-KK-SSH
whoami
cd /var/log
EOF-KK-SSH